Readme.md

```diff
@@ -249,11 +250,15 @@ import { AdSettings } from 'react-native-fbads';
 
 AdSettings contains global settings for all ad controls.
 
+#### deviceHashedId
```

Comment:
You're right, maybe it does not have much sense to try to be consistent with their names.

Will change to `currentDeviceHash `.